client/src/commands.ts

```diff
@@ -74,6 +75,7 @@ export function registerCommands(
   const commands: Command[] = [
     restartNgServer(client),
     openLogFile(client),
+    getTemplateTcb(client),
```

Comment:
One thing I noticed here is that the other commands in this list don't depend on the Angular LS being initialized yet, but this one does. We might want to consider _not_ registering this command right away, but instead keep a separate list of commands that we register once the initialization is complete.